lib/prometheus/business_collector.rb

```diff
@@ -38,6 +38,12 @@ def initialize
     @client_systems = PrometheusExporter::Metric::Gauge.new(
       'client_systems', 'Systems from clients (excludes Red Hat)'
     )
+    @total_systems_by_os = PrometheusExporter::Metric::Counter.new(
+      'total_systems_by_os', 'Systems by OS version'
+    )
+    @client_systems_by_os = PrometheusExporter::Metric::Counter.new(
+      'client_systems_by_os', 'Systems by OS version (excluded Red Hat)'
+    )
```

Comment:
@skateman I'm quite sure we should update this to be a Gauge. From [the docs](https://prometheus.io/docs/concepts/metric_types/#counter):

> A counter is a cumulative metric that represents a single monotonically increasing counter whose value can only increase or be reset to zero on restart. For example, you can use a counter to represent the number of requests served, tasks completed, or errors.
> 
> Do not use a counter to expose a value that can decrease. For example, do not use a counter for the number of currently running processes; instead use a gauge.

I've updated it in https://github.com/RedHatInsights/compliance-backend/pull/932